autokeras/adapters/input_adapter.py

```diff
@@ -191,3 +191,78 @@ def infer_column_types(self):
         for key, value in column_types.items():
             if key not in self.column_types:
                 self.column_types[key] = value
+
+
+class TimeSeriesInputAdapter(adapter_module.Adapter):
+
+    def __init__(self,
+                 lookback=None,
+                 column_names=None,
+                 column_types=None,
+                 **kwargs):
+        super().__init__(**kwargs)
+        self.lookback = lookback
+        self.column_names = column_names
+        self.column_types = column_types
+
+    def get_config(self):
+        config = super().get_config()
+        config.update({
+            'lookback': self.lookback,
+            'column_name': self.column_names,
```

Comment:
`column_names` (missing s). Please make sure you have appropriate tests in order to catch such issues.

Reply:
@abgese Would you please add a file `tests/node_test.py` to use these two functions to test the config functions of the nodes? https://github.com/keras-team/autokeras/blob/master/autokeras/graph.py#L35-L45 You may find how to use these two functions in other parts of the code. Thank you!

Reply:
Done